src/content/manifest/environment-variables.mdx

```diff
@@ -0,0 +1,104 @@
+---
+title: Environment variables
+description: Use Okteto's built-in environment variables to streamline your Okteto manifest.
+sidebar_label: Environment variables
+id: environment-variables
+---
+
+The Okteto manifest utilizes environment variables with each section supporting specific ENV VARs. Additional ENV VARs are added regularly and it's important to understand the existing ones and their usage. Currently, there are three types of ENV VARs to be aware of:
+
+## Using environment variables
+
+In order to use environment variables within your Okteto manifest you must add them to `$OKTETO_ENV` within your `deploy` section. This will make the environment variables available in all following manifest sections (e.g. `destroy`, `dev`) as well as in the Okteto UI. To do this you’ll add something similar to this in your `deploy` section (using your actual variable names):
```

Comment:
`Okteto manifest you must add them to $OKTETO_ENV` -> this is not a hard requirement. You can use env vars wihout using `OKTETO_ENV`. `OKTETO_ENV` is to configure dynamic endpoints for external resources